src/ui/Forms/Main.cs

```diff
@@ -29560,7 +29554,7 @@ public void GotoPrevSubPosFromvideoPos()
         {
             if (mediaPlayer.VideoPlayer != null && _subtitle != null)
             {
-                double ms = mediaPlayer.VideoPlayer.CurrentPosition * TimeCode.BaseUnit;
+                double ms = Math.Round(mediaPlayer.CurrentPosition * TimeCode.BaseUnit, MidpointRounding.AwayFromZero);
```

Comment:
The round here is because I've had cases where the video position is at something like 39.999999 and the start time is 40, and since it's ms, I don't see a need for the numbers after the decimal point.